sound/soc/sof/amd/acp.h

```diff
@@ -139,6 +139,17 @@ struct acp_dsp_stream {
 	unsigned int reg_offset;
 };
 
+struct sof_amd_acp_desc {
+	unsigned int rev;
+	unsigned int host_bridge_id;
+	unsigned int pgfsm_base;
+	unsigned int ext_intr_stat;
+	unsigned int dsp_intr_base;
+	unsigned int sram_pte_offset;
+	unsigned int i2s_pin_config_offset;
+	unsigned int hw_semaphore_offset;
```

Comment:
you may want to use u32 for the offsets, to avoid any ambiguity on what 'int' means.

Reply:
Done